lib/addDefaults.js

```diff
@@ -10,6 +10,53 @@ var findAccessorMinMax = require('./findAccessorMinMax');
 
 module.exports = addDefaults;
 
+/**
+ * Adds default values to any missing glTF values of the hierarchy.
+ *
+ * @param {Object} gltf - An object holding a glTF hierarchy
```

Comment:
Even though Cesium doesn't use the hyphens I kind of like them here. @pjcozzi do you have a preference?